taylor-francis/taylor-francis-proposal-2020-07.md

```diff
@@ -0,0 +1,308 @@
+# Building Research Software
+
+-   Damien Irving
+-   Kate Hertweck
+-   Luke Johnston
+-   Sara Mahallati
+-   Joel Ostblom
+-   Charlotte Wickham
+-   Greg Wilson
+
+## 1. Aims
+
+Software is now as essential to research as telescopes, test tubes, and
+reference libraries, which means that researchers need need to know how to
+build, check, use, and share programs.  However, most introductions to
+programming focus on developing commercial applications, not on exploring
+problems whose answers aren't yet known.  Our goal is show readers how to do
+that, both on their own and as part of a team.
+
+We believe every researcher should know how to write short programs that clean
+and analyze data in a reproducible way, and how to use version control to keep
+track of what they have done.  But just as some astronomers spend their careers
+designing telescopes, some researchers focus on building the software that makes
+research possible.  People who do this are called *research software engineers*,
+and the aim of this book is to get you ready for this role, i.e., to help you go
+from writing code for yourself to creating tools to help your entire field
+advance.
+
+## 2. Synopsis
+
+This book introduces the core skills researchers need to develop robust software
+that others can use, to share their work with others, and to be productive as
+part of a research software team.  We assume readers have some basic programming
+knowledge, and build on that to cover:
+
+-   using the Unix shell to manage work and make it repeatable
+-   effective use of version control
+-   building reusable software tools
+-   automating workflows
+-   configuring software
+-   testing and error handling
+-   creating productive, inclusive teams
+-   publishing in a reproducible way
```

Comment:
```suggestion

```
Delete if proposal in #415 adopted.

Reply:
We could add "documenting, packaging and releasing software" as the second last bullet point (above "creating productive, inclusive teams" which comes after packaging in the #415 proposal).

Reply:
:+1: